src/diversity/index.tsx

```diff
@@ -0,0 +1,163 @@
+import { ButtonLink } from '@components/button';
+import Head from 'next/head';
+import SayHi from 'src/index/say-hi';
+import Layout from 'src/layout';
+import style from './diversity.module.css';
+
+const Diversity = () => {
+  return (
+    <Layout title="Variant – Mangfold">
+      <Head>
+        <meta
+          property="og:description"
+          content="Vi i Variant samarbeider med de som aktivt jobber for å skape mangfold, fordi det gir verdi og nytte for absolutt alle."
+        />
+        <meta
+          name="description"
+          content="Vi i Variant samarbeider med de som aktivt jobber for å skape mangfold, fordi det gir verdi og nytte for absolutt alle.."
+        />
+      </Head>
+      <SayHi
+        className={style.sayHi}
+        href="https://twitter.com/intent/tweet?screen_name=variant_as"
```

Comment:
kan vi fjerne linken til Twitter først som sist? Ingen er på Twitter lengre 😅 Kan vi heller linke til instagram/messenger/mail?

Reply:
Veldig fair, foreslår å legge inn IG i stedet, men tror det er twitter overalt ellers også. Kanskje best å sette opp en egen branch/PR der det gjøres overalt?

Reply:
Tror hele denne kan fjernes for dette, også på variant.no. Egen PR er fornuftig. Tenkte litt på om vi kunne lage noe morsomt ut av det integrert mot Slack eller noe heller, men det får bli en annen gang

Reply:
(uten at jeg er enig i at _ingen_ er på Twitter da)

Reply:
Det her går under det vi har begynt å diskutere rundt IA. Kan det ikke gjøres om til «Bli en Variant»? Dagens menypunkt kan da erstattes med én av de nye sidene som kommer.

Reply:
Tror det fungerer greit både å fjerne og å bytte til Bli en Variant, men lar den ligge som det er her så den er consistent med det andre, så tas det heller i egen PR 👍

Reply:
Jeg ville ha fjernet det fra denne siden da (ansattsiden har det f.eks heller ikke). Og heller hatt egen PR for å fjerne det var fremsiden